SimG4CMS/Calo/src/CaloTrkProcessing.cc

```diff
@@ -52,45 +55,61 @@ CaloTrkProcessing::CaloTrkProcessing(const std::string& name,
   DDsvalues_type sv(fv.mergedSpecifics());
 
   G4String value = "Calorimeter";
-  std::vector<std::string> caloNames = getNames(value, sv);
+  std::vector<std::string> caloNames = getNames(value, sv, false);
```

Comment:
better to capture these return values as `const std::vector<std::string>&` (similarly for all other calls to `getNames()` or `getNumbers()` in this file)